code/script.js

```diff
@@ -1,51 +1,199 @@
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/zoltar.jpg" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight;
+  chat.scrollTop = chat.scrollHeight
+}
+
+
+// function to swap from one question to the next
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
+
+  if (questionNumber === 1) {
+    userReply(message)
+    input.value = ''
+    setTimeout(() => fortuneYesNo(message), 1000)
+  } else if (questionNumber === 2) {
+    userReply(message)
+    setTimeout(() => showFortune(message), 1000)
+  } else if (questionNumber === 3) {
+    userReply(message)
+    setTimeout(() => chosenFortune(message), 1000)
+  } else if (questionNumber === 4) {
+    userReply(message)
+    setTimeout(() => showPrice(message), 1000)
+    input.value = ''
+  } else {
+    userReply(message)
+    setTimeout(thankYou(message), 1000)
+  }
 }
 
 // Starts here
-const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+const greeting = () => {
+  questionNumber = 1
+  botReply(`Greetings child, tell me your name`)
+}
+
+const fortuneYesNo = (msg) => {
+  questionNumber++
+  botReply(`Well met, ${msg}! Will you tell me of your deepset desires?`)
+  // had to insert introsound here due to browsers blocking .play on page load without user interaction
+  throatsing.play()
+
+  inputWrapper.innerHTML = `
+    <button id="yesBtn">Yes</button>
+    <button id="noBtn">No</button>
+  `
+
+  document
+    .getElementById('yesBtn')
+    .addEventListener('click', () => nextQuestion('yes'))
+  document
+    .getElementById('noBtn')
+    .addEventListener('click', () => nextQuestion('no'))
 }
 
-// Set up your eventlisteners here
+const showFortune = (type) => {
+  questionNumber++
+
+  if (type === 'yes') {
+
+     botReply ('So be it, choose wisely')
+
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>Tell me of...</option>
+        <option value="future">my future</option>
+        <option value="family">the fate of my family</option>
+        <option value="..wait, I just want to be big">Wait, I think I just want to be big</option>
+      </select>
+    `
+  } else {
+    botReply('Very well, be gone then.'),
+    setTimeout(location.reload(), 3000)
+  }
+
+  const select = document.getElementById('select')
+  select.addEventListener('change', () => nextQuestion('Tell me of my ' + select.value))
+}
+
+const chosenFortune = (type) => {
```

Comment:
The function chosenFortune does not seem to be working.

Reply:
Hmmm, having a hard time making it work, I'll push the latest changes. I had a capital F on one and just f on another "future" but even though i changed that I cant seem to get it to work... any ideas what could be the problem?

Reply:
I think it might be the comparing string in (type === ' '). 
Try `(type === "Tell me of my future") ` `(type === "Tell me of my family")`  because that's the parameter we're passing to nextQuestion().